wheel/src/api.rs

```diff
@@ -159,20 +164,25 @@ pub fn get_puzzle_and_solution_for_coin<'a>(
                 result,
                 &Coin::new(find_parent, find_ph, find_amount),
             ) {
-                Err(ValidationErr(n, _)) => Err(EvalErr(n, "coin not found".to_string())),
+                Err(ValidationErr(n, _)) => {
+                    Err(EvalErr::InvalidOpArg(n, "coin not found".to_string()))
+                }
                 Ok(pair) => Ok(pair),
             }
         })
-        .map_err(|e| {
-            let blob = node_to_bytes(&allocator, e.0).ok().map(hex::encode);
-            PyValueError::new_err((e.1, blob))
-        })?;
+        .map_err(|e| map_pyerr_w_ptr(&e, &allocator))?;
 
     // keep serializing normally, until wallets support backrefs
     let serialize = node_to_bytes;
     Ok((
-        PyBytes::new(py, &serialize(&allocator, puzzle)?),
-        PyBytes::new(py, &serialize(&allocator, solution)?),
+        PyBytes::new(
+            py,
+            &serialize(&allocator, puzzle).map_err(|e| map_pyerr_w_ptr(&e, &allocator))?,
+        ),
+        PyBytes::new(
+            py,
+            &serialize(&allocator, solution).map_err(|e| map_pyerr_w_ptr(&e, &allocator))?,
```

Comment:
all of these explicit conversions to `PyErr` is making it look pretty good to have a `python-binding` build feature on `clvm_rs` that implements `From<EvalErr> for PyErr`

Reply:
Because I need to make it a toggleable feature to avoid breaking wasm